src/mame/hp/hp9825.cpp

```diff
@@ -51,6 +45,7 @@
 #include "sound/beep.h"
 #include "hp9825.lh"
 #include "softlist_dev.h"
+#include "machine/ram.h"
```

Comment:
Please group the headers, and sort alphabetically within groups:
* Prefix header first
* Local project headers (e.g. hp9825_tape.h and hp9825_optrom.h)
* Stuff from the devices library
* Stuff from the emu library
* Generated layout source